tests/test_cheatdetection.py

```diff
@@ -0,0 +1,47 @@
+"""Tests challengeutils.cheat-detection functions"""
+# pylint: disable=line-too-long
+from unittest import mock 
+
+import synapseclient
+
+from challengeutils import cheat_detection
+
+
+
+class TestCheatDetection:
+    def setup(self):
+        """Setup test"""
+
+        self.syn = mock.create_autospec(synapseclient.Synapse)
+
+        self.evaluation_id = "123456"
+        
+        self.cheat_detection_obj = cheat_detection.CheatDetection(
+            syn=self.syn,
+            evaluation_id=self.evaluation_id
+        )
+        
+
+    def test_string_representation(self):
+        "Tests string representation of Cheat Detection"
+        assert str(self.cheat_detection_obj) == """
+Evaluation ID: 123456
+Accepted Submissions: 0
+Potentially Linked Users: 0
+        """
+    
+
+    def test_evaluation_id(self):
+        "Test that the evaluation id is valid"
+        assert self.cheat_detection_obj.evaluation=="123456"
```

Comment:
Nit: add whitespace around `==`

Reply:
It works with Awesome Challenge now. It wasn't able to handle challenges with one submission, now it can.